vertx-pojo-mapper-common/src/main/java/de/braintags/io/vertx/pojomapper/dataaccess/query/impl/FieldCondition.java

```diff
@@ -12,6 +12,9 @@
  */
 package de.braintags.io.vertx.pojomapper.dataaccess.query.impl;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import org.apache.commons.lang3.StringUtils;
```

Comment:
Blocking what? I just use them for the toString if the value is a list. Doing it manually, I would still have to iterate over the list, which is also blocking.